include/nbl/builtin/hlsl/prefix_sum_blur/box_sampler.hlsl

```diff
@@ -0,0 +1,183 @@
+#include "nbl/builtin/hlsl/cpp_compat.hlsl"
+#include "nbl/builtin/hlsl/enums.hlsl"
+#include "nbl/builtin/hlsl/macros.h"
+
+#ifndef _NBL_BUILTIN_BOX_SAMPLER_INCLUDED_
+#define _NBL_BUILTIN_BOX_SAMPLER_INCLUDED_
+
+namespace nbl
+{
+namespace hlsl
+{
+namespace prefix_sum_blur
+{
+
+template<typename PrefixSumAccessor, typename T>
+struct BoxSampler
+{
+    using prefix_sum_accessor_t = PrefixSumAccessor;
+
+    PrefixSumAccessor prefixSumAccessor;
+    uint16_t wrapMode;
+    uint16_t linearSize;
+    T normalizationFactor;

```

Comment:
its always tied and derived from radius, don't even make it a parameter, work it out inside `operator()~`